exe/arduino_ci.rb

```diff
@@ -424,16 +388,16 @@ def perform_unit_tests(cpp_library, file_config)
   platforms = choose_platform_set(config, "unittest", config.platforms_to_unittest, cpp_library.library_properties)
   if platforms.empty?
     explain_and_exercise_envvar(VAR_EXPECT_UNITTESTS, "unit tests", "platforms and architectures") do
-      puts "    Configured platforms: #{config.platforms_to_unittest}"
-      puts "    Configuration is default: #{config.is_default}"
+      @log.iputs "    Configured platforms: #{config.platforms_to_unittest}"
```

Comment:
I guess these could be wrapped with `@log.indent`.

Reply:
Good catch, thanks!  Refactoring whitespace is surprisingly difficult 😅